src/Map.jsx

```diff
@@ -75,6 +76,24 @@ class Map extends Component {
     );
   }
 
+  getStopInfo(route, stopCoordinates) {
+    let stops = this.state.selectedStops;
+    const index
```

Comment:
@EddyIonescu 
I saw your comment about using static, but it's not allowed in strict mode
So I just removed the function and put the reference in here

Reply:
Since we're not using `index` anywhere else, could we just used `find` or `first` instead?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find